packages/patternfly-4/react-core/src/components/Tabs/Tabs.d.ts

```diff
@@ -0,0 +1,16 @@
+import { SFC, HTMLProps, FormEvent } from 'react';
+import { Omit } from '../../typeUtils';
+
+export interface TabsProps extends Omit<HTMLProps<HTMLDivElement>, 'onSelect'> {
+  children: any;
```

Comment:
children included in html props

Reply:
You should import this and add this to the typescript integration app since it's more complicated component.